mmv1/third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -128,11 +152,27 @@ resource "google_logging_metric" "logging_metric" {
 `, suffix, filter)
 }
 
+func testAccLoggingMetric_loggingBucket(suffix string, filter string, project_id string) string {
+	return fmt.Sprintf(`
+resource "google_logging_metric" "logging_metric" {
+  name        = "my-custom-metric-%s"
+  bucket_name = "projects/%s/locations/global/buckets/_Default"
+  filter      = "%s"
+
+  metric_descriptor {
+  	metric_kind = "DELTA"
+    unit        = "1"
+    value_type  = "INT64"
+  }
+}
+`, suffix, project_id, filter)
+}
+
 func testAccLoggingMetric_descriptionUpdated(suffix, description string) string {
 	return fmt.Sprintf(`
 resource "google_logging_metric" "logging_metric" {
 	name        = "my-custom-metric-%s"
-	description = "Counter for  VM instances that have hostError's"
```

Comment:
Not relevant strictly for this patch, but hope it's ok to still delete since it's a minor thing